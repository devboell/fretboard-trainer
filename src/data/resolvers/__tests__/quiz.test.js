import knex from '../../connector'
import allQuizzes from '../quiz'

describe('resolver index', () => {
  beforeEach(async () => {
    await knex.migrate.rollback()
    await knex.migrate.latest()
    await knex.seed.run()
  })

  afterEach(async () => {
    await knex.migrate.rollback()
  })

  afterAll(() => {
    knex.destroy()
  })

  it('findAll', async () => {
    const quizzes = await allQuizzes()
    expect(quizzes).toMatchSnapshot()
  })
})
