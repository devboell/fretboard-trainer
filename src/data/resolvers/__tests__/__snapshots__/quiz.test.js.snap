// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolver index findAll 1`] = `
Array [
  Object {
    "id": 1,
    "name": "quiz one",
  },
  Object {
    "id": 2,
    "name": "quiz two",
  },
  Object {
    "id": 3,
    "name": "quiz three",
  },
  Object {
    "id": 4,
    "name": "quiz four",
  },
  Object {
    "id": 5,
    "name": "quiz five",
  },
  Object {
    "id": 6,
    "name": "quiz six",
  },
]
`;
